pkg/utils/kubernetes/kubernetes.go

```diff
@@ -79,6 +79,19 @@ func CreateOrUpdate(ctx context.Context, c client.Client, obj client.Object, f c
 	return controllerutil.CreateOrUpdate(ctx, c, obj, f)
 }
 
+func ResolveSecrets(ctx context.Context, client client.Client, secretRefs []v1alpha1.ObjectReference) ([]corev1.Secret, error) {
```

Comment:
can you add a description about what the function is doing.

Reply:
Done